doc/_static/css/nrfxlib.css

```diff
@@ -1,3 +1,80 @@
 :root {
   --docset-color: #ff9933;
 }
+
+/* Doc link box on the ZBOSS page */
+
+.rst-content blockquote.doc-link {
+ padding: 0 10px;
+ display: block;
+ width: 450px;
+ overflow:hidden;
+ transition: all 0.35s ease;
+ box-sizing: border-box;
+ line-height: 50px;
+ vertical-align:middle;
+ position:relative;
+ margin-left: 0;
```

Comment:
pls format CSS using two spaces indent (as root entry)

Reply:
And done, in a separate commit.